admin/attributes_controller.php

```diff
@@ -2157,17 +2157,17 @@ function update_option(theForm) {
         document.body.style.cursor = "wait";
 
         // set initial values
-        var SelectedOption = theForm.options_id.options[theForm.options_id.selectedIndex].value;
-        var theField = document.getElementById("OptionValue");
+        let SelectedOption = theForm.options_id.options[theForm.options_id.selectedIndex].value;
+        let theField = document.getElementById("OptionValue");
 
         // reset the array of pulldown options so it can be repopulated
-        var Opts = theField.options.length;
+        let Opts = theField.options.length;
         while (Opts > 0) {
-          Opts = Opts - 1;
+          Opts -= 1;
           theField.options[Opts] = null;
         }
 
-<?= zen_js_option_values_list('SelectedOption', 'theField') ?>
+<?php echo zen_js_option_values_list('SelectedOption', 'theField'); ?>
```

Comment:
Why the change to full `echo` syntax?

Reply:
poor little IDE couldn't cope with that in the middle of javascript so was flagging two errors.